lib/teracy-dev-essential/config/rsync_recovery.rb

```diff
@@ -110,6 +121,14 @@ def valid_command(plugins)
             exec "vagrant up"
           end
         end
+
+        rsync_cmd = ARGV.any? { |s| s.include?('rsync-auto') }
+
+        if rsync_cmd and Vagrant::Util::Platform.linux? and TeracyDev::Util.require_version_valid?(Vagrant::VERSION, "< 2.2.3")
```

Comment:
if rsync_cmd and TeracyDev::Util.require_version_valid?(Vagrant::VERSION, "< 2.2.3")